app/script/main/app.js

```diff
@@ -191,10 +195,9 @@ z.main.App = class App {
       client: new z.client.ClientService(this.auth.client, storageService),
       connect: new z.connect.ConnectService(this.auth.client),
       connectGoogle: new z.connect.ConnectGoogleService(this.auth.client),
-      conversation: z.util.Environment.browser.edge
-        ? new z.conversation.ConversationServiceNoCompound(this.auth.client, storageService)
-        : new z.conversation.ConversationService(this.auth.client, storageService),
+      conversation: new z.conversation.ConversationService(this.auth.client, eventService, storageService),
       cryptography: new z.cryptography.CryptographyService(this.auth.client),
+      event: eventService,
```

Comment:
You don't like to define it inline? No real need for a const

Reply:
Humm the thing is I cannot because I need to inject it in another service also (https://github.com/wireapp/wire-webapp/pull/4312/files#diff-1b79670fa81e80094c3d8329873acd39R198)